coreos-base/cros-devutils/cros-devutils-9999.ebuild

```diff
@@ -31,6 +31,7 @@ RDEPEND="cros_host? ( app-emulation/qemu )
 	dev-util/shflags
 	cros_host? ( dev-util/crosutils )
 	app-crypt/efitools
+	cros_host ? ( cros-base/coreos-sb-keys )
```

Comment:
coreos-devel/sdk-depends is a better place for this. Also when you do that bump the revision (`-r%d` in the filename) of the ebuild (typically just maintained as a symlink instead of renaming the real file)

For reference there is also a coreos-base/hard-host-depends ebuild that serves a similar purpose to coreos-devel/sdk-depends but it has a weird name.